.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,47 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "30"
```

Comment:
This means fetch recent 30 commits for get changed files.
In usually, a PR request commits times low that this number.